Sources/Apodini/Components/Handler/Handler.swift

```diff
@@ -6,7 +6,7 @@
 //
 
 import NIO
-#if compiler(>=5.4) && $AsyncAwait
+#if compiler(>=5.5) && $APODINI_EXPERIMENTAL_ASYNC_AWAIT
```

Comment:
Does it need the dollar sign? afaik you just pass a active compilation condition via `-DAPODINI_EXPERIMENTAL_ASYNC_AWAIT` and I never used a dollar sign for those 🙃 so just checking, seems unfamiliar to me.